### PR TITLE
feat(cli): add mage command to update golden integration test files

### PR DESCRIPTION
**Pull request by DmitriyLewen**
## Description
Add mage command to update golden integration test files

## Related PRs
- [x] #4379
- [x] #4404

## Checklist
- [x] I've read the [guidelines for contributing](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/) to this repository.
- [x] I've followed the [conventions](https://aquasecurity.github.io/trivy/latest/community/contribute/pr/#title) in the PR title.
- [ ] I've added tests that prove my fix is effective or that my feature works.
- [ ] I've updated the [documentation](https://github.com/aquasecurity/trivy/blob/main/docs) with the relevant information (if needed).
- [ ] I've added usage information (if the PR introduces new options)
- [ ] I've included a "before" and "after" example to the description (if the PR is a user interface change).

### CODE REVIEW

File: integration/testdata/conda-spdx.json.golden [hunk 1]
```diff
@@ -1,113 +1 @@
...
...opyrightText":"","externalRefs":[{"referenceCategory":"PACKAGE-MANAGER","referenceType":"purl","referenceLocator":"pkg:conda/pip@22.2.2"}],"primaryPackagePurpose":"LIBRARY","files":[{"fileName":"miniconda3/envs/testenv/conda-meta/pip-22.2.2-py38h06a4308_0.json","SPDXID":"SPDXRef-File-7eb62e2a3edddc0a","checksums":[{"algorithm":"SHA1","checksumValue":"a6a2db7668f1ad541d704369fc66c96a4415aa24"}],"copyrightText":""}]},{"name":"testdata/fixtures/fs/conda","SPDXID":"SPDXRef-Filesystem-6e0ac6a0fab50ab4","downloadLocation":"NONE","copyrightText":"","attributionTexts":["SchemaVersion: 2"],"primaryPackagePurpose":"SOURCE"}],"relationships":[{"spdxElementId":"SPDXRef-DOCUMENT","relatedSpdxElement":"SPDXRef-Filesystem-6e0ac6a0fab50ab4","relationshipType":"DESCRIBES"},{"spdxElementId":"SPDXRef-Filesystem-6e0ac6a0fab50ab4","relatedSpdxElement":"SPDXRef-Application-ee5ef1aa4ac89125","relationshipType":"CONTAINS"},{"spdxElementId":"SPDXRef-Application-ee5ef1aa4ac89125","relatedSpdxElement":"SPDXRef-Package-b1088cb4090e3a55","relationshipType":"CONTAINS"},{"spdxElementId":"SPDXRef-Application-ee5ef1aa4ac89125","relatedSpdxElement":"SPDXRef-Package-6b677e82217fb5bd","relationshipType":"CONTAINS"}]}
```
Comment (knqyf263) [inline integration/testdata/conda-spdx.json.golden:1]: Can we format this golden file?

**Review by knqyf263 — COMMENTED** [review]
LGTM. I left a nitpicking comment.

File: integration/testdata/conda-spdx.json.golden (hunk 1 above)
Comment (DmitriyLewen) [reply integration/testdata/conda-spdx.json.golden:1]: I created #4404 for that.

Comment (DmitriyLewen) [reply integration/testdata/conda-spdx.json.golden]: I have updated this file in https://github.com/aquasecurity/trivy/pull/4380/commits/b15b5b167303c3bbda1a4b21b37f898622dbf2c6

**Review by knqyf263 — APPROVED** [review]

**Pull request merged**